### DISCUSSION TITLE
CocList mapping not work.

### DISCUSSION THREAD

**Discussion by jackhub** [neoclide/coc.nvim#3643] [Q&A] [answered]
I use this mapping to put coclist outline to right window, so I can see more infos.
But those commands `<c-w>L60<c-w><bar>i` not got executed.

nnoremap <space>2               `m0:<C-u>CocList --normal outline<cr><c-w>L60<c-w><bar>i`

Please take a look.

**Comment by chemzqm** [comment, marked as answer]
It won't work since `:CocList` use notification, and  the list is supposed to be used for quick pick you should use `:CocOutline` instead.
